modules/ROOT/pages/mule-upgrade-tool.adoc

```diff
@@ -1,15 +1,17 @@
 = Mule Upgrade Tool
 
-*Latest version: 1.1.0*
+*Latest version: 1.1.1*
```

Comment:
```suggestion
```

Reply:
This won't be ready from the release and probably isn't needed because the latest version is clear from https://docs.mulesoft.com/release-notes/mule-upgrade-tool/mule-upgrade-tool, which is maintained as part of each version release.

Reply:
ok, I'll remove it

Reply:
done